tests/integration/kubernetes/tests_common.sh

```diff
@@ -37,3 +37,32 @@ get_pod_config_dir() {
 	pod_config_dir="${BATS_TEST_DIRNAME}/runtimeclass_workloads"
 	info "k8s configured to use runtimeclass"
 }
+
+# Generate kubernetes service yaml from template
+k8s_generate_service_yaml() {
+    local service_yaml="${1}"
+    local image="${2}"
+
+    local fixtures_dir="${BATS_TEST_DIRNAME}/fixtures"
+
+    # Extract name from the file name
+    local name=$(basename "${service_yaml%.*}")
+
+    local service_yaml_template="${fixtures_dir}/service.yaml.in"
+  
+    NAME="${name}" IMAGE="${image}" RUNTIMECLASS="kata" \
+        envsubst < "${service_yaml_template}" > "${service_yaml}"
+}
```

Comment:
As you're adding it, please, do the following:
* Add this on its own commit
* Make sure to use it in all tests that are using annotations
* Only then use it as part of the test being added.

Reply:
I can add a new commit with this change. But no other test is using this function except the confidential test. This is because other tests are taking a different approach and are reading yaml files directly from the `runtimeclass_workloads` dir.

Reply:
Then, please, change this test to do exactly what the other tests are doing, and read files from the `runtimeclass_workloads` directory.

If we have a pattern, like this one please, let's follow the pattern rather than introduce a new one and then we have to deal with 2 or more patterns in the future.

Reply:
Okay, instead of generating the yaml file for every test run, I am now following what has been done for other tests. I am just adding a file under `runtimeclass_workloads` which will be used for this test.